examples/testsuites/simple.py

```diff
@@ -1,27 +1,33 @@
 from lisa import CaseMetadata, SuiteMetadata
-from lisa.common.logger import log
 from lisa.core.testSuite import TestSuite
+from lisa.util.logger import log
 
 
 @SuiteMetadata(area="demo", category="simple", tags=["demo"])
 class SimpleTestSuite(TestSuite):
     @CaseMetadata(priority=1)
-    def hello(self):
-        log.info("hello world")
+    def hello(self) -> None:
+        log.info("environment: %s", len(self.environment.nodes))
+        default_node = self.environment.defaultNode
+        result = default_node.execute("echo hello world!")
+        log.info("stdout of node: '%s'", result.stdout)
```

Comment:
We should probably swap to f-strings throughout the codebase, as % formatting is prone to errors.